model/src/main/kotlin/config/S3FileStorageConfiguration.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2017 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+/**
+ * A class to hold the configuration for using local files as a storage.
+ */
+data class S3FileStorageConfiguration(
+    /**
+     * The name of the S3 bucket used to store files in.
+     */
```

Comment:
You may want to use more compact one-liners like

    /** The name of the S3 bucket used to store files in. */

to document properties.

Also, consider adding empty lines to separate the properties, so it's visually more clear which comment belongs to which property.

Reply:
Addressed. Although, there are many other files in the codebase that still use three lines of comment per field.